__tests__/app.test.js

```diff
@@ -311,3 +311,118 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles?sort_by=...&order=...", () => {
```

Comment:
This describe can also be combined/nested into the previous "GET /api/articles" describe